.taskcluster.yml

```diff
@@ -41,6 +41,8 @@ tasks:
             - "-cx"
             - "curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py &&
                rustup component add clippy rustfmt &&
+               rustup toolchain install nightly &&
```

Comment:
Maybe we should use the `rustlang/rust:nightly` image instead of `rust:buster` so we don't have to install nightly in the task?

Reply:
Nightly clippy lints and fmt formatting are different from the ones found by stable version

Reply:
Is nightly required for cargo udeps?

Reply:
Yes https://github.com/est31/cargo-udeps#usage